lib/Transform/XTenMinimizeLiveTensors.cpp

```diff
@@ -281,7 +281,12 @@ class XTenMinimizeLiveTensorsPass
       return llvm::any_of(op.getOperands(), [&](Value value) {
         // okay so long as it doesn't use the output of a scheduled op
         auto *defOp = value.getDefiningOp();
-        return defOp != nullptr && opToInfo.find(defOp) != opToInfo.end();
+        if (defOp != nullptr && opToInfo.find(defOp) != opToInfo.end()) {
+          llvm::errs() << "Unknown operation: ";
```

Comment:
```suggestion
          llvm::errs() << "Illegal operation: ";
```
Thinking again, maybe it would be better to output `Illegal operation`